src/EnergyPlus/SolarShading.cc

```diff
@@ -7194,6 +7194,10 @@ namespace SolarShading {
 			} else {
 				SUN3( DayOfYear, AvgSinSolarDeclin, AvgEqOfTime );
 				AvgCosSolarDeclin = std::sqrt( 1.0 - pow_2( AvgSinSolarDeclin ) );
+				// trigger display of progress in the simulation every two weeks
```

Comment:
@mjwitte and @Myoldmopar I could have put this trigger to show the date (by setting DisplayPerfSimulationFlag to true) just about anywhere but it parallels the location of the "Updating Shadowing Calculations" in the enclosing if block. Do you think this is a good place to put this?